NetSuiteProject/Exercise.js

```diff
@@ -2,22 +2,65 @@
 function add(x,y){
 	return x + y;
 };
+
+function addition(x,y){
+	if(typeof add (x,y) === "string"){
+	alert("This function only accepts Integers")
+	}
+	else {return x + y;}
```

Comment:
You will need to actually make these changes in the code and push them on the feature branch so that the pull request updates accordingly.